vpr/src/route/rr_graph_reader.cpp

```diff
@@ -15,6 +15,9 @@
 
```

Comment:
This should probably be two files now? One for the XML version and one for the flatbuffer version?